static/js/panel.js

```diff
@@ -118,8 +118,10 @@ $(function() {
     }
 
     // If the user toggles the checkbox let's update the state accordingly.
-    $('#panel-accel-enable')[0].addEventListener('change', () => {
-        var newState = ($('#panel-accel-enable')[0].checked);
+    let panelAccelEnable = $('#panel-accel-enable')[0];
```

Comment:
Given how we're using it, we may as well just use `document.getElementById('panel-accel-enable')`, if you want.